content/4.examples/query-payment-states.md

```diff
@@ -0,0 +1,138 @@
+# Query payment states
+
+Corda stores the state of a payment in the vault of the node that initiated the payment. So you can access the state of a payment by querying the vault.
+
+```mermaid
+classDiagram
+  class MyPaymentService {
+    + queryPaymentsViaGevamu()
+  }
+  class GevamuFacade {
+    + getPayment()
+    + queryPayments()
+  }
+
+  MyPaymentService ..> GevamuFacade
+```
+
+## Payment structure
+
+Structure of the [`Payment`](https://gevamu.github.io/corda-payments-sdk/payments-contracts/com.gevamu.corda.states/-payment/index.html) is as follows:
+
+::notice{info}
+  Pay attention, that full payment instruction is stored as Corda [`Attachment`](https://docs.r3.com/en/api-ref/corda/4.8/open-source/javadoc/net/corda/core/contracts/Attachment.html). So you need to download it and deserialize it to get all payment details.
```

Comment:
```suggestion
 Note that the full payment instruction is stored as a Corda [`Attachment`](https://docs.r3.com/en/api-ref/corda/4.8/open-source/javadoc/net/corda/core/contracts/Attachment.html). To get all payment details, download the attachment and deserialize it.
```

Reply:
I would like to save casual relationship between sentences. Consequences of storing instructions as attachment are the reason, why it is inside `notice`